src/api/DataSetInterface.ts

```diff
@@ -65,4 +64,4 @@ export interface IAllocateDataSetOption {
 }
 
 // When DataSetUtilsV2 for zos-node-accessor v2 is ready, alias DataSetUtilsV2 to DataSetUtils.
-export { DataSetUtils as DataSetUtils };
+// export { DataSetUtils as DataSetUtils };
```

Comment:
`DataSetUtils` works with `zos-node-accessor` v1 only. `DataSetInterface` is independent to `zos-node-accessor` version. So I export `DataSetInterface` only, while `DataSetUtils` plays as the internal one. 

When we move to `zos-node-accessor` v2, we'll have new `DataSetUtilsV2`, replacing `DataSetUtils` in `DataSetInterface`. The code using `DataSetInterface` will not be impacted.

Reply:
That makes sense.
Could we try to come up with another way of aliasing it (once zos-node-accessor v2 is releases), to avoid circular dependencies?

Reply:
Maybe we can re-export DataSetUtilsV2 in `index.ts`?

```
export { DataSetUtilsV2 as DataSetUtils } from "./DataSetUtilsV2";
```

Reply:
that should work.
We can do that when it's time to adopt zos-node-accessor v2 :yum: